services/provider/server/server.go

```diff
@@ -817,6 +877,35 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 
 }
 
+func getExternalResourceLabels(kind string, isMirroringReplicationEnabled bool, isFlattenMode bool,
+	replicationID string, storageID string) map[string]string {
+	labels := make(map[string]string)
+	switch kind {
+	case "VolumeReplicationClass", "VolumeGroupReplicationClass":
+		if isMirroringReplicationEnabled {
+			replicationID := replicationID
```

Comment:
I think this is redundant

Reply:
This is the flag for setting up those IDs